src/prefect/agent/agent.py

```diff
@@ -109,6 +110,17 @@ class Agent:
             and all deployed flow runs
     """
 
+    # Loop intervals for query sleep backoff
+    _loop_intervals = {
```

Comment:
is it common for users to write their own agent classes? I'm wondering how likely it is we'd break calls to `Agent.start` that provide `_loop_intervals` as an arg

Reply:
afaik it is not a supported thing. That argument is "private" and exposed just for testing, we generally don't want users to be hitting the API at a different set of intervals.